src/components/pages/UserPage.js

```diff
@@ -31,10 +32,36 @@ export const UserPage = ({ match }) => {
     }
   }, []);
 
+  const onSortChange = (event) => {
+    const sortValue = event.target.value;
+    setSortType(sortValue);
+  };
+
+  const SelectDropdown = () => (
```

Comment:
Can we move this to be its own file and component? It could be shared between the userPage and the similar dropdown for commit list later